frontend/src/services/hooks.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * Commonly used hooks
+ */
+
+import React from "react";
+
+import { Observable } from "rxjs";
+
+/**
+ * hooks that are used repeatedly
+ */
+class HooksService {
```

Comment:
Do not put custom hooks into service. There's no point putting it into a class as you use each functions separately. Create separate module for hooks e.g. `src/hooks` and rename this file to `observable.tsx` and then just export `useObservable` function

Reply:
Done